style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+}
+
+.card__img {
+  display: flex;
+  justify-content: center;
+  padding: 32px 16px 0;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__text {
+  display: flex;
+  align-self: center;
+  padding-top: 40px;
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  padding: 4px 16px 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+/* #region StarsRegion */
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images\star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
```

Comment:
Please [correct order of css properties](https://9elements.com/css-rule-order/). Fix it everywhere

Reply:
Done